pandas/core/common.py

```diff
@@ -2672,6 +2672,9 @@ def is_list_like(arg):
      return (hasattr(arg, '__iter__') and
             not isinstance(arg, compat.string_and_binary_types))
 
+def is_named_tuple(arg):
+    return isinstance(arg, tuple) and hasattr(arg, '_fields')
```

Comment:
Ah cool!
Lots of discussion on those boards about a better way of doing `namedtuple`. It's not perfect at the moment, but it's very functional (we use them a lot - maybe too much)